src/Calculator.Shared/Views/NumberPad.xaml

```diff
@@ -117,14 +117,13 @@
             string coming from the RESW file
         -->
 		<controls:CalculatorButton x:Name="decimalSeparatorButton"
-															 x:Uid="decimalSeparatorButton"
-															 Grid.Row="4"
-															 Grid.Column="2"
-															 Style="{Binding ElementName=ControlRoot, Path=ButtonStyle}"
-															 Background="{ThemeResource AppBackgroundAltMediumLowBrush}"
-															 AutomationProperties.AutomationId="decimalSeparatorButton"
-															 ButtonId="Decimal"
-															 IsEnabled="{Binding IsDecimalEnabled}" />
-															 <!-- UNO TODO common:KeyboardShortcutManager.Character="."-->
+								   x:Uid="decimalSeparatorButton"
+								   Grid.Row="3"
+								   Grid.Column="2"
+								   Style="{x:Bind Path=DecimalButtonStyle, Mode=OneWay}"
```

Comment:
Why does it need its own style?

Reply:
Because it's neither a numeric button style, nor a operator button style. Here is the Windows 10 Calculator. The dot is Bold and font equals to 24, like the numeric buttons, but it also has a grey background, like the operator button.
![image](https://user-images.githubusercontent.com/11750340/58436170-ff19b980-8091-11e9-8aaa-da5cee686f70.png)

Reply:
how did it work?

Reply:
If I summarize what you explained me Mathieu, it should work as it is but my style is a just workaround.
We suggest to merge this and fix this (and remove the workaround later).
Can you confirm that, @davidjohnoliver ?

Reply:
I'll take @MatFillion 's word as I haven't looked closely enough, only thing I would add is that the button font size seems to change depending on view state (portrait vs landscape etc)

Reply:
cc @MaximeDionNventiveCom 
I've just tested this (on iOS and UWP) and it is working as the following :
Standard & Currency : I didn't see any changes
Programmer : Indeed the FontSize change on iOS and something strange happens on UWP (see image at the end)
Scientific : It not working well on iOS at all but Maxime Dion is on it. Looks good on UWP.

![image](https://user-images.githubusercontent.com/11750340/58442837-005fdc00-80bc-11e9-8985-5b9e6cda4a91.png)